index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href=" style.css">
+        <title>Simple Calculator</title>
+    </head>
+    <body>
+        <div id='main'>
```

Comment:
Use better descriptive class names. Instead of `main`, a more descriptive class name could be `main_calculator_container`